cmake/modules/FindMayaUsd.cmake

```diff
@@ -50,7 +50,8 @@ find_package_handle_standard_args(MayaUsd
     REQUIRED_VARS
         MAYAUSD_INCLUDE_DIR
         MAYAUSDAPI_LIBRARY
-        MAYAUSD_MOD_PATH
+        #MAYAUSD_MOD_PATH is not part of MayaUsd cuts downloaded so remove it from the requirements
```

Comment:
This won't be meaningful to external developers.  I would just remove this comment.

Reply:
I would maybe put a comment on line 11 (where this var is described) saying its optional